src/components/Login.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+
+// login component
+function Login() {
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+  const [error, setError] = useState(null);
+
+  function isValidEmail(email) {
+    return /\S+@\S+\.\S+/.test(email);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    alert("Submitted");
+
+    if (isValidEmail(email)) {
+      console.log("email is valid");
+    } else {
+      setError("email is invalid");
+    }
+  };
```

Comment:
Let's use [react hook form](https://react-hook-form.com/) to manage form state and validation.

Let's also use yup for input validation

Check this post on how react hook form is used 
https://dev.to/franciscomendes10866/react-form-validation-with-react-hook-form-and-yup-4a98